content/commands/hpttl/index.md

```diff
@@ -0,0 +1,87 @@
+---
+acl_categories:
+- '@read'
+- '@hash'
+- '@fast'
+arguments:
+- display_text: key
+  key_spec_index: 0
+  name: key
+  type: key
+- display_text: numfields
+  name: numfields
+  type: integer
+- display_text: field
+  multiple: true
+  name: field
+  type: string
+arity: -4
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+command_flags:
+- readonly
+- fast
+complexity: O(N) where N is the number of arguments to the command
+description: Returns the TTL in milliseconds of a hash field.
+group: hash
+hidden: false
+key_specs:
+- RO: true
+  access: true
+  begin_search:
+    spec:
+      index: 1
+    type: index
+  find_keys:
+    spec:
+      keystep: 1
+      lastkey: 0
+      limit: 0
+    type: range
+linkTitle: HPTTL
+since: 8.0.0
+summary: Returns the TTL in milliseconds of a hash field.
+syntax_fmt: HPTTL key FIELDS numfields field [field ...]
+syntax_str: FIELDS numfields field [field ...]
+title: HPTTL
+---
+Like [`HTTL`]({{< relref "/commands/httl" >}}) this command returns the remaining TTL (time to live) of a field that has an
+expiration set, but in milliseconds instead of seconds.
+
+The return value in case of error is one of:
```

Comment:
Should remove either "the return value" from this line or "the command returns" from the list items. Probably remove from the list items to be consistent with the other pages? (Same issue with HTTL, btw.)